### PR TITLE
Revert "Borg Upgrades"

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Reverts Aurorastation/Aurora.3#6395 due to #6400.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/life.dm [hunk 1]
```diff
@@ -19,7 +19,6 @@
 		process_killswitch()
 		process_locks()
 		process_queued_alarms()
-		process_level_restrictions()
```
Comment (Sindorman) [inline code/modules/mob/living/silicon/robot/life.dm]: This line should be removed liked you did

File: code/modules/mob/living/silicon/robot/drone/drone.dm [hunk 2]
```diff
@@ -249,8 +249,6 @@
 	return 1
 
 //DRONE LIFE/DEATH
-/mob/living/silicon/robot/drone/process_level_restrictions()
```
Comment (Sindorman) [inline code/modules/mob/living/silicon/robot/drone/drone.dm]: Put the code back

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -1016,12 +1016,8 @@
 							to_chat(cleaned_human, "<span class='warning'>[src] cleans your face!</span>")
 		return
 
-/mob/living/silicon/robot/proc/self_destruct(var/anti_theft=FALSE)
```
Comment (Sindorman) [inline code/modules/mob/living/silicon/robot/robot.dm]: Put the code back

**Comment by MarinaGryphon** [comment]
This is just a simple revert of the PR to prevent *critical* issues such as cyborgs exploding. You can do those changes on your own branch.

**Comment by TehFlaminTaco** [comment]
This is a fantastic anti-theft method. 10/10

**Comment by Sindorman** [comment]
You can disable critical bug by removing only one line I put on review. You don't have to revert it fully

**Comment by Arrow768** [comment]
Closes this in favor of #6403 as this is a bugfix and not a revert

**Pull request closed without merging**